src/oc_erchef/apps/chef_db/src/chef_db.erl

```diff
@@ -167,6 +166,7 @@ create(ObjectRec0, #context{server_api_version = ApiVersion, reqid = ReqId}, Act
     Fields = chef_object:fields_for_insert(ObjectRec2),
     case stats_hero:ctime(ReqId, {chef_sql, create_object},
                           fun() -> chef_sql:create_object(QueryName, Fields) end) of
+        {ok,[[{<<"add_user">>,<<>>}]]} -> ok;
```

Comment:
This is very specific to the add_user query/function.  If we are not able to get a return value consistent with other inserts (number of rows inserted), it's worth a closer look at epgsql+sqerl to understand the  returned `[[{binary(), binary()}]]` from stored proc calls. Would it be sufficient to match similar to the below w/ guards: